app/assets/javascripts/admin/utils/services/status_message.js.coffee

```diff
@@ -5,8 +5,8 @@ angular.module("admin.utils").factory "StatusMessage", ->
       alert:    {style: {color: 'grey'}}
       notice:   {style: {color: 'grey'}}
       success:  {style: {color: '#9fc820'}}
-      failure:  {style: {color: '#da5354'}}
-      error:   {style: {color: '#da5354'}}
+      failure:  {style: {color: '#C85136'}}
+      error:   {style: {color: '#C85136'}}
```

Comment:
I wonder if we could define CSS variables... but probably not worth it at the moment. So this does change some colours even without the feature toggle on.